### PR TITLE
Switch back to default jquery ui CSS

### PR DESCRIPTION
**Pull request by deming7h777**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ const blastScriptConfig = {
         { from: path.join(nodeModules, '/datatables.net-colreorder-dt/css/colReorder.dataTables.css'), to: blastStyles},
         { from: path.join(nodeModules, '/datatables.net-bs/css/dataTables.bootstrap.css'), to: blastStyles},
         { from: path.join(nodeModules, '/datatables.net-dt/css/jquery.dataTables.css'), to: blastStyles},
+        { from: path.join(nodeModules, '/jquery-ui-1.11.1/jquery-ui.css'), to: blastStyles},
```
Comment (hsiaoyi0504) [inline webpack.config.js]: You need to add **jquery-ui** into **package.json** and remove the **jquery-ui.css**.

File: blast/templates/blast/main.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 <div class="tripal_details">
 
-<form method="post" id="MainBlastForm" name="MainBlastForm" enctype="multipart/form-data">
+<form method="post" id="MainBlastForm" name="MainBlastForm" enctype="multipart/form-data" autocomplete="on">
```
Comment (hsiaoyi0504) [inline blast/templates/blast/main.html]: It looks like this change is not related to this PR.

File: blast/templates/blast/main.html [hunk 3]
```diff
@@ -175,7 +175,7 @@
         </fieldset>
 
         <fieldset id="fieldset-options-blast-filter-param">
-        <legend>Filters and Masking</legend>
+        <legend>Filters and Masking</legend>s
```
Comment (hsiaoyi0504) [inline blast/templates/blast/main.html]: Please remove the typo here.

File: webpack.config.js (hunk 1 above)
Comment (deming7h777) [reply webpack.config.js]: package2
as far as i can see , it's already iclude jquery in jason.package should i add the extra "jquery-ui"?

Comment (hsiaoyi0504) [reply webpack.config.js]: Based on this line, you install **jquery-ui.css ** from another npm package called **jquery-ui-1.11.1**.

Comment (deming7h777) [reply webpack.config.js]: i find out that there is a jquery-ui.css in juqery-ui-themes but its version is 1.12.0, i will also figure it if i need to install the version 1.11.1 or just use 1.12.0

**Comment by hsiaoyi0504** [comment]
Another thing is that **.gitignore** need to be updated.

**Comment by hsiaoyi0504** [comment]
It would be good to make a separate commit for updating the request version. Then, rebase toward master as well.

**Comment by deming7h777** [comment]
ok, i got it but the commit of "requests update" is already in the other issue which is waiting for review should i have the same commit here?

**Comment by deming7h777** [comment]
travis ci always failed first and somehow passed after the several rebuild

**Comment by hsiaoyi0504** [comment]
It depends on which PR will be merged first. I would suggest separate that here. The duplicate changes/commits will be removed later (during the rebase).

**Comment by deming7h777** [comment]
@hsiaoyi0504 , @childers  ok i have separated these commit can u review it again
i think this PR can be murge already

**Comment by hsiaoyi0504** [comment]
You haven't added the updated **package-lock.json** back.

**Comment by hsiaoyi0504** [comment]
Also, rebase toward master branch if you can.

**Comment by childers** [comment]
@deming7h777 Any updated on @hsiaoyi0504 's comment?

**Comment by g8tor** [comment]
Closing because this was directly implemented in g8tor/javascript-updates branch
